src/App.jsx

```diff
@@ -1,17 +1,38 @@
+import { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
+import messages from './data/messages.json';
 
 const App = () => {
+  const [entries, setEntries] = useState(
+    messages.map((entry) => ({...entry}))
+  );
```

Comment:
@Shirley425, not sure as to why you chose to `.map` over the messages and convert them to an array. You could have simply initiated state from `messages` directly, like so.
```suggestion
  const [entries, setEntries] = useState(messages);
```